QuoridorEngine/src/core/QuoridorGame.cs

```diff
@@ -58,9 +58,28 @@ public bool IsTerminalState()
         /// </summary>
 		/// <param name="playerIsWhite">True if we want moves for white player, false otherwise</param>
         /// <returns>A list of all the possible moves from this state for given player</returns>
-        public List<Move> GetPossibleMoves(bool playerIsWhite)
+        public IEnumerable<Move> GetPossibleMoves(bool playerIsWhite)
         {
-            throw new NotImplementedException();
+            List<QuoridorMove> possibleMoves = new();
```

Comment:
Are you sure that this correctly initializes a new List<QuoridorMove> and not just a random type? Please double check if not sure to avoid hard to track bugs